backfill.sh

```diff
@@ -132,7 +132,7 @@ set +o nounset; . install.sh; set -o nounset;
 
         if [ ! -f $xmlpath ] && [ ! -f $xmlunpubpath ]; then
             # lax doesn't know where the remote location and it's not on the fs
-            if [ $remotepath = "no-location-stored" ]; then
+            if [ "$remotepath" == "no-location-stored\r" ]; then
```

Comment:
why the `/r`?
When I run
```
elife@prod--lax:/srv/lax$ IFS=,
elife@prod--lax:/srv/lax$ ./manage.sh --skip-install report all-article-versions-as-csv | head | while read msid version remotepath; do echo "PATH: BEGIN${remotepath}END"; done                                                                            
PATH: BEGINhttps://raw.githubusercontent.com/elifesciences/elife-article-xml/6b463b2b469748cbc03b571122377cc9191cd7aa/articles/elife-00003-v1.xmlEND
PATH: BEGINhttps://raw.githubusercontent.com/elifesciences/elife-article-xml/6b463b2b469748cbc03b571122377cc9191cd7aa/articles/elife-00005-v1.xmlEND
PATH: BEGINhttps://raw.githubusercontent.com/elifesciences/elife-article-xml/944c3f97d1153a08ab23a1860c30e1177895df9e/articles/elife-00007-v1.xmlEND
```
There seem to be no `\r` in `remotepath`, but I can't find an example with `no-location-stored` now.